woocommerce/woocommerce
# Clean up 'Profile Wizard' code

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/50782

Profile wizard was replaced by Core Profiler last year, this PR remove the old profile wizard code.

Changes:

- Remove `plugins/woocommerce-admin/client/profile-wizard` directory
- `client/profile-wizard/steps/usage-modal.js` is also used in `WCPay/UsageModal.js`, so it's moved to `./client/task-lists/components/usage-modal.js`. We can further refactor `WCPay/UsageModal.js` file to merge the code but I don't want to make that change in this PR to keep it simple.
- Remove `ProfileWizard` from controller.js and fix setting chunk name
- Clean up css related to profile wizard
- Remove hook reference for `woocommerce_admin_profile_wizard_steps` as it's not used anymore



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fresh site with this branch
2. Go through the Core Profiler and make sure it works as expected and UI is displayed correctly




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
